searches/linear_search.jl

```diff
@@ -0,0 +1,13 @@
+function search(array, key)
+    for i in 1:length(array)
+        if array[i] == key
+            return i
+        end
+    end
+    return -1
+end
+
+array = [1,3,4,7,8,11]
+println(search(array, 3))
+println(search(array, 8))
+println(search(array, 12))
```

Comment:
Hmm, probably this should be `11`, as it displays `-1`.
```suggestion
println(search(array, 11))
```

Reply:
The point of this line is to show what the function returns if the key is not found.